Issue 508: Horizontal review updates to security considerations (#510)
w3c/wcag2ict: security-considerations.md

## Security Considerations

> This Working Group Note does not introduce any new security considerations. However, when implementing WCAG 2 success criteria in the context of non-web ICT, accessibility features could potentially be exploited to bypass software security features. It is best practice to choose implementations that reduce the potential for unauthorized access (e.g. ability to bypass logins and CAPTCHAs) or introduction of malicious software by cyberattackers. 

Reviewer: Might it be worth saying something like:

> However, when implementing WCAG 2 success criteria in the context of non-web ICT, accessibility features, as with any additional features, could potentially be exploited to bypass software security features.

This just highlights that it isn't a unique aspect of accessibility features but a challenge with adding _any_ additional features.

Author: The latest updates include the WAI proposed text as-is (from the Google doc).

> This Working Group Note does not introduce any new security considerations. However, when implementing WCAG 2 success criteria in the context of non-web ICT, accessibility features could potentially be exploited to bypass software security features. It is best practice to choose implementations that reduce the potential for unauthorized access (e.g. ability to bypass logins and CAPTCHAs) or introduction of malicious software by cyberattackers. 

Reviewer 2: ```suggestion
This Working Group Note does not introduce any new security considerations. Since any software feature has the potential to compromise security, take care when implementing non-web ICT features added to meet WCAG 2 success criteria.
```
Suggestions as per WAI Team input.

<div class="note">
  

> The WCAG 2 Security Considerations section also notes specific success criteria that are identified to have possible implications for security that could also exist for non-web ICT.</div>

Author: ```suggestion
The [WCAG 2 Security Considerations section](https://www.w3.org/TR/WCAG22/#security-summary) lists specific success criteria with possible implications for security, which could also exist for non-web ICT..</div>
```
